server/src/graphql-types/ChapterUser.ts

```diff
@@ -21,4 +21,7 @@ export class ChapterUser {
 
   @Field(() => User)
   user: User;
+
+  @Field(() => Boolean)
```

Comment:
```suggestion
  @Field(() => Boolean, { nullable: true })
```
Otherwise we'd be implying this exists every time that `ChapterUser` is returned and that's not always true.

It's a pretty minor deal for booleans, but I think we should for the sake of consistency, if nothing else.